src/services/PullService.ts

```diff
@@ -0,0 +1,94 @@
+import { SyncPullQuery } from "../queries/SyncPullQuery";
+import { Inject, Service, Token } from "typedi";
+import { InjectRepository } from "typeorm-typedi-extensions";
+import { Pull } from "../db/entities/Pull";
+import { Repository } from "typeorm";
+import { ILoggerToken } from "../common/global";
+import { timeALaterThanTimeB } from "../utils/util";
+import { encodeLabelArray } from "../utils/parser";
+import { Logger } from "probot";
```

Comment:
Please split the imported library and the imported own file.

Reply:
good idea.